template/src/global.css

```diff
@@ -16,6 +16,15 @@
   /* Build Onchain Apps shared variables */
   --boat-color-background: var(--palette-bg-black);
   --boat-color-text: var(--palette-bg-white);
+  --boat-color-blue-40: #1354e1;
```

Comment:
Why are we adding them here? If we are using them only on Tailwind we can keep things clean and just have the Tailwind config.